Admin/AbstractAdmin.php

```diff
@@ -2382,6 +2398,13 @@ public function setBaseCodeRoute($baseCodeRoute)
      */
     public function getBaseCodeRoute()
     {
+        if ($this->isChild()) {
+            return $this->getParent()->getBaseCodeRoute().'|'.$this->getCode();
+        }
```

Comment:
Anyway, I realized that this will not work. I will go for something like this:
```php
// NEXT_MAJOR: Uncomment the following lines.
// if ($this->isChild()) {
//     return $this->getParent()->getBaseCodeRoute().'|'.$this->getCode();
// }
//
// return $this->getCode();

// NEXT_MAJOR: Remove all the code above.
if ($this->isChild()) {
    $parentCode = $this->getParent()->getCode();

    if ($this->getParent()->isChild()) {
        $parentCode = $this->getParent()->getBaseCodeRoute();
    }

    return $parentCode.'|'.$this->getCode();
}

return $this->baseCodeRoute;
```